app/models/service_template_ansible_playbook.rb

```diff
@@ -52,6 +52,15 @@ def self.create_catalog_item(options, auth_user)
     end
   end
 
+  def self.manager_valid?(manager)
```

Comment:
The quickest way actually is to use `manager.configuration_script_payloads.empty?`. It selects only 1 record at most.